webrender/src/platform/macos/font.rs

```diff
@@ -69,23 +70,47 @@ fn supports_subpixel_aa() -> bool {
     data[0] != data[1] || data[1] != data[2]
 }
 
-fn get_glyph_metrics(ct_font: &CTFont, glyph: CGGlyph) -> GlyphMetrics {
+fn get_glyph_metrics(ct_font: &CTFont,
+                     glyph: CGGlyph,
+                     x_subpixel: SubpixelOffset,
```

Comment:
it looks like the pair of x/y sub-pixel offsets is repeated often, so it's begging to be wrapped into a struct. I don't see an appropriate `euclid` type, so just declaring a new struct should be fine, since you aren't going to do any operations with the struct itself anyway.